EMULATOR/EMULATOR.C

```diff
@@ -1972,13 +1972,19 @@ void Emulator_UpdateVRAM()
 
 void Emulator_BlitVRAM()
 {
+	if (activeDispEnv.isinter)
+	{
+		//Emulator_Clear(0, 0, activeDispEnv.disp.w, activeDispEnv.disp.h, 128, 128, 128);
+		return;
+	}
+
 	Emulator_SetTexture(vramTexture, TF_16_BIT);
 	Emulator_SetShader(g_blit_shader);
 
-	u_char l = word_33BC.disp.x / 8;
-	u_char t = word_33BC.disp.y / 8;
-	u_char r = word_33BC.disp.w / 8 + l;
-	u_char b = word_33BC.disp.h / 8 + t;
+	u_char l = activeDispEnv.disp.x / 8;
```

Comment:
Good, but I would also add a comment where activeDispEnv is declared saying "//word_33BC"